apps/jubmoji-quest/src/pages/collect/index.tsx

```diff
@@ -163,13 +162,17 @@ export default function CollectJubmojiPage() {
         jubmojiCollectionCards,
         sig.pubKeyIndex
       );
+      await addJubmoji(realJubmoji);
```

Comment:
Unrelated to this PR, but do we have a ticket to create a UX for collection when users already have that Jubmoji in their wallet?

Reply:
#77